Formula/dune.rb

```diff
@@ -16,9 +16,10 @@ class Dune < Formula
   depends_on "ocaml" => [:build, :test]
 
   def install
-    system "ocaml", "bootstrap.ml"
-    system "./dune.exe", "build", "-p", "dune", "--profile", "dune-bootstrap"
-    bin.install "_build/default/bin/dune.exe" => "dune"
+    system "./configure", "--mandir=#{man}"
```

Comment:
Does this effect projects which use `dune`? The wording here makes me wonder that:

> `DIR where man pages are installed are for the default build context`

Reply:
Not sure, tbh. However, they make a big deal about `dune` being [relocatable](https://github.com/ocaml/dune#installation), so I would guess otherwise, as this would make `dune` very much not relocatable.

Reply:
That it shows up in the `dune` binary is a bit worrying tho.

I guess I can let it install the man directory where it wants to initially (i.e. directly into `prefix`) and move it afterward just to be on the safe side.

Reply:
Yeah, moving is what Alpine and Arch do.

Watch out for `doc` as well as `man`.